examples/cpp_control_interface/.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
Funny that the auto new line is not working for you. Maybe you can check you vs code, it should insert new lines at the end of file automatically. I also configured it now to skip additional ones.

Reply:
Can we mark this as resolved?